examples/example.krun

```diff
@@ -59,3 +59,7 @@ SKIP=[
 ]
 
 N_EXECUTIONS = 2  # Number of fresh processes.
+
+# No. of seconds to wait before taking the initial temperature reading.
+# You should set this hight enough for the system to cool down a bit.
+TEMP_READ_PAUSE = 1
```

Comment:
This particular config file is supposed to be a fast-running example to get user going. See below regarding the default value however.